### PR TITLE
SDA-7644: (Follow up) Use Version.ID instead of Telemetry Openshift Version for HS OS Version (rosa describe output)

### PR DESCRIPTION
**Pull request by davidleerh**

### CODE REVIEW

File: cmd/describe/cluster/cmd.go [hunk 1]
```diff
@@ -138,6 +138,13 @@ func run(cmd *cobra.Command, argv []string) {
 		)
 	}
 
+	var openshiftVersion string
+	if cluster.Hypershift().Enabled() {
```
Comment (gdbranco) [inline cmd/describe/cluster/cmd.go:142]: Would check if OpenShiftVersion is empty alongside this

File: cmd/describe/cluster/cmd.go [hunk 2]
```diff
@@ -138,6 +138,13 @@ func run(cmd *cobra.Command, argv []string) {
 		)
 	}
 
+	var openshiftVersion string
```
Comment (gdbranco) [inline cmd/describe/cluster/cmd.go:141]: Could just be openshiftVersion := cluster.OpenshiftVersion()

File: cmd/describe/cluster/cmd.go [hunk 3]
```diff
@@ -138,6 +138,13 @@ func run(cmd *cobra.Command, argv []string) {
...
+	if cluster.Hypershift().Enabled() {
+		openshiftVersion = cluster.Version().ID()
+	} else {
```
Comment (gdbranco) [inline cmd/describe/cluster/cmd.go:144]: Would not be needed if suggestions above are taken into consideration

**Comment by davidleerh** [comment]
Hi @gdbranco, Thank you for your review! :)
I went ahead and made the changes as requested. Please kindly let me know if there are additional amendments required.

**Comment by pvasant** [comment]
@davidleerh @gdbranco can we merge this?

**Comment by davidleerh** [comment]
Hi @pvasant, I believe we are currently awaiting PM approval on how to proceed:
Screenshot from 2023-01-19 12-10-04

Hi @etabak, would you kindly advise if this change can be merged into ROSA since it only affects "rosa describe cluster" version output only? Or should we wait for additional PM approval?

**Comment by tzvatot** [comment]
> Hi @pvasant, I believe we are currently awaiting PM approval on how to proceed: Screenshot from 2023-01-19 12-10-04
> 
> Hi @etabak, would you kindly advise if this change can be merged into ROSA since it only affects "rosa describe cluster" version output only? Or should we wait for additional PM approval?

Please do not merge this until / if we get an ack from PM.
I would like to avoid this confusion - these are two different fields with two different meanings.

**Comment by tzvatot** [comment]
Per PM, closing this one as not needed.

**Comment by gdbranco** [comment]
Any news on state for this?

**Comment by davidleerh** [comment]
Hi @gdbranco,
I believe there will be a future task to pursue having metrics populate the value for HS clusters (cluster.OpenshiftVersion()).
The idea of making changes in ROSA CLI was dismissed; therefore, closing PR.

**Pull request closed without merging**